app/uk/gov/hmrc/helptosavefrontend/controllers/email/AccountHolderUpdateEmailAddressController.scala

```diff
@@ -85,7 +85,7 @@ class AccountHolderUpdateEmailAddressController @Inject() (val helpToSaveService
     val nino = htsContext.nino
 
     if (emailVerificationParams.nino =!= nino) {
-      auditor.sendEvent(SuspiciousActivity(nino, "nino_mismatch"), nino)
+      auditor.sendEvent(SuspiciousActivity(None, s"nino_mismatch, expected=$nino, received=${emailVerificationParams.nino}"), nino)
```

Comment:
Pls change logger.warn string to "SuspiciousActivity: email was verified but nino [${emailVerificationParams.nino}] in URL did not match user's nino"